docs/sql/system.txt

```diff
@@ -677,6 +677,85 @@ deactivate the stats tracking. This will also wipe all existing log entries::
     reset the value of ``stats.enabled`` to its default which is false.
 
 
+Settings checks
+===============
+
+The table ``sys.checks`` contains misconfigured settings of a cluster.
```

Comment:
more precise: e.g. the table exposes a list of internal cluster checks and their validation results